.github/workflows/testing.yml

```diff
@@ -9,6 +9,10 @@ on:
       - master
   pull_request:
 
+env:
+  PUPPETEER_SKIP_CHROMIUM_DOWNLOAD: 1
+  PUPPETEER_EXECUTABLE_PATH: /usr/bin/google-chrome
```

Comment:
What are these lines? Are you able to use the Chromium that comes from within Ubuntu by default??! If this is the case, the `yarn install` will end up being a lot faster. 

One criticism is that this magic is so far away from where it matters. And that disconnect makes the code harder to understand in the future maybe. Perhaps the simplest solution would be to not have it here at the root but instead put it right before where it matters and gets used. E.g.

```
      - name: Check Yarn install diffs
        env:
           # This tells 'yarn install' that when it installs puppeteer, it doesn't need to 
           # download a copy of chromium.
           PUPPETEER_SKIP_CHROMIUM_DOWNLOAD: 1
        run: |
          yarn install --ignore-scripts
```

Basically, anything so that the future you and me can more easily debug this in 6 months when we've forgotten all about how this stuff works.

Reply:
Yeah, I hoisted it up to the top to avoid figuring out all the parts where it might be needed, but I can try pushing the download one to the install steps, and the executable to the test run

Reply:
I added the SKIP to the other install calls and found the `development.yml` also uses puppeteer, so added it there too